### ISSUE TITLE
Not possible to see bottom parts of the left-side menu of the Testing page

### ISSUE THREAD

**Issue by martinhbramwell** [awatson1978/starrynight#73]
When viewing the StarryNight web site on a small screen (or a small window on a big screen) there is no scrolling action possible to see the parts of the left-side menu cut off by the bottom of the window.

```
     website/client/app/components/testing/testingSidebar
```

**Comment by awatson1978** [comment]
Yup, that makes sense.  I have the code to fix that in other apps; and have been extracting a `clinical:active-layout` package that's going to have a consistent approach to sidebars, scrolling, and whatnot.
